src/main.h

```diff
@@ -133,6 +133,89 @@
 #define STAR_RATING_REJECTED 0x274C //Unicode Character 'Cross Mark'
 #define STAR_RATING_STAR 0x2738 //Unicode Character 'Heavy Eight Pointed Rectilinear Black Star'
 
+#define GQ_ICON_ADD "list-add"
+#define GQ_ICON_REMOVE "list-remove"
+#define GQ_ICON_UNDO "edit-undo"
+#define GQ_ICON_OPEN "document-open"
+#define GQ_ICON_SAVE "document-save"
+#define GQ_ICON_SAVE_AS "document-save-as"
+#define GQ_ICON_NEW "document-new"
+#define GQ_ICON_EDIT "document-edit"
+#define GQ_ICON_REVERT "document-revert"
+#define GQ_ICON_CLOSE "window-close"
+#define GQ_ICON_RUN "system-run"
+#define GQ_ICON_STOP "process-stop"
+#define GQ_ICON_FULLSCREEN "view-fullscreen"
+#define GQ_ICON_LEAVE_FULLSCREEN "view-restore"
+#define GQ_ICON_REFRESH "view-refresh"
+#define GQ_ICON_ABOUT "help-about"
+#define GQ_ICON_QUIT "application-exit"
+#define GQ_ICON_DELETE "edit-delete"
+#define GQ_ICON_DELETE_SHRED "edit-delete-shred"
+#define GQ_ICON_CLEAR "edit-clear"
+#define GQ_ICON_COPY "edit-copy"
+#define GQ_ICON_FIND "edit-find"
+#define GQ_ICON_REPLACE "edit-find-replace"
+#define GQ_ICON_PRINT "document-print"
+
+#define GQ_ICON_GO_TOP "go-top"
+#define GQ_ICON_GO_BOTTOM "go-bottom"
+#define GQ_ICON_GO_UP "go-up"
+#define GQ_ICON_GO_DOWN "go-down"
+#define GQ_ICON_GO_FIRST "go-first"
+#define GQ_ICON_GO_LAST "go-last"
+#define GQ_ICON_GO_PREV "go-previous"
+#define GQ_ICON_GO_NEXT "go-next"
+#define GQ_ICON_GO_JUMP "go-jump"
+#define GQ_ICON_HOME "go-home"
+#define GQ_ICON_PAN_DOWN "pan-down"
+#define GQ_ICON_PAN_UP "pan-up"
+
+#define GQ_ICON_PREV_PAGE "media-skip-backward"
+#define GQ_ICON_NEXT_PAGE "media-skip-forward"
+#define GQ_ICON_BACK_PAGE "media-seek-backward"
+#define GQ_ICON_FORWARD_PAGE "media-seek-forward"
+
+#define GQ_ICON_PLAY "media-playback-start"
+#define GQ_ICON_PAUSE "media-playback-pause"
+
+#define GQ_ICON_ZOOM_IN "zoom-in"
+#define GQ_ICON_ZOOM_OUT "zoom-out"
+#define GQ_ICON_ZOOM_100 "zoom-original"
+#define GQ_ICON_ZOOM_FIT "zoom-fit-best"
+
+// might need replacing
+#define GQ_ICON_PREFERENCES "preferences-system"
+#define GQ_ICON_HELP "help-contents" // "help-browser"?
+#define GQ_ICON_EXPORT "document-export" // use collection icon?
+
+// not available in some themes?
```

Comment:
Since "dialog-cancel" and "emblem-ok" are not listed in [Standard Icon Names](https://specifications.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html#names) there is little difference with "gtk-apply", "gtk-cancel" and "gtk-ok".
Gtk recommends replacing these icons with labels "_Apply", "_Cancel" and "_OK".

Reply:
I know, it is a problem with several stock icons not having a suggested replacement in standard icon names. I think that in most use cases the labels are already present (although sometimes they are different, like "Start"). I didn't have a good solution.

Reply:
What about following:
1) leave label if button already has it, remove icon
2) use suggested label if button doesn't have any
3) use some other icon if it applicable in context, e.g. "media-playback-start" icon for "Start" button

Reply:
I have no strong feelings on the matter. I just tried to preserve all the icons that were present as closely as possible. 

Generally I suppose if this bunch of commits gets merged, it will be possible to fine-tune the look. My preference would probably be also to remove some of the icons as I prefer cleaner look.